File: js/estrutura-sequencial/retangulo.js
const readline = require('readline-sync');

const base = Number(readline.question("Base do retangulo: "));
const altura = Number(readline.question("Altura do triangulo: "));

const area = base * altura;
const perimetro = 2 * (base * altura);
const diagonal = Math.sqrt((base * base) + (altura * altura));

console.log(`AREA = ${area}`);
console.log(`PERIMETRO = ${perimetro}`);
console.log(`DIAGONAL = ${diagonal.toFixed(4)}`);

File: c++/retangulo.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;
int main() {

    double base, altura, area, perimetro, diagonal;

    cout << "Base do retangulo: ";
    cin >> base;
    cout << "Altura do retangulo: ";
    cin >> altura;

    area = base * altura;
    perimetro = 2 * (base + altura);
    diagonal = sqrt(powf(base, 2) + powf(altura, 2));

    cout << "AREA = " << fixed << setprecision(4) << area << endl;
    cout << "PERIMETRO = " << fixed << setprecision(4) << perimetro << endl;
    cout << "DIAGONAL = " << fixed << setprecision(4) << diagonal << endl;

    return 0;
}